src/ui/hooks/useTransactionHook.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from 'react';
+
+import { useTransactionStore } from '@/ui/stores/transactionStore';
+import { useWallet } from '@/ui/utils/WalletContext';
+
+// Temporary fix before getting the coinlist tokenlist upgrade in the background.
+export const useTransactionHook = () => {
+  const usewallet = useWallet();
```

Comment:
The react convention is `const wallet = useWallet()` as in `const location = useLocation()`
```suggestion
  const wallet = useWallet();
```